jsonnet/telemeter/client/kubernetes.libsonnet

```diff
@@ -244,6 +244,25 @@ local securePort = 8443;
                 record: 'cluster:telemetry_selected_series:count',
                 expr: 'max(federate_samples - federate_filtered_samples)',
               },
+              {
+                expr: |||
+                  sum by (namespace) (
+                    rate(federate_requests_failed_total{job="telemeter-client"}[15m])
+                  ) /
+                  sum by (namespace) (
+                    rate(federate_requests_total{job="telemeter-client"}[15m])
+                  ) > 0.2
```

Comment:
Why 20% as the threshold? Checking similar alarms with `ag -G '\.*' 'failed_total' jsonnet/` I see PrometheusRemoteStorageFailures, AlertmanagerClusterFailedToSendAlerts and AlertmanagerFailedToSendAlerts with 1% threshold. Arguably, those alarms are more critical, but anyway 20% looks quite high, what is the rationale behind that value?

Reply:
Nevermind, I saw Simon's previous comment